modules/custom/az_paragraphs/css/az_paragraphs_az_text_media.css

```
@@ -27,11 +27,14 @@
   background: none;
 }
 .box {
-  padding: 2em;
-  margin: 3em 0;
+  padding-left:2em;
+  padding-right: 2em;
```

Oops, box alters margin so we should probably just leave padding alone at 2em
```suggestion
  padding:2em;
```